app.js

```diff
@@ -92,7 +93,15 @@ app.use((err, req, res, next) => {
   logger.error(err, 'burped error');
   res.status(err.status || 500).json({msg: err.message});
 });
+
 const httpServer = app.listen(PORT);
+process.on('uncaughtException', function(err) {
+    if(err.errno === 'EADDRINUSE' && useMultiple) {
```

Comment:
I'm not sure 'useMultiple' is needed.  It seems completely redundant with the information provided the `JAMBONES_IGNORE_EADDRINUSE` env.  Or, to put it another way, if someone configured the system with `JAMBONES_IGNORE_EADDRINUSE` isn't that enough to tell us that this new code needs to be executed?

Reply:
correct. I removed it.